app/src/main/java/com/dropbox/android/sample/Graph.kt

```diff
@@ -94,8 +96,11 @@ object Graph {
                         source?.let { adapter.fromJson(it) }
                     }.getOrNull()
                 },
-                writer = { _, _ ->
+                writer = { _, config ->
                     val buffer = Buffer()
+                    withContext(Dispatchers.IO) {
```

Comment:
Are you sure you need the `withContext` here? If you need it, isn't better to include the `write` call in the `withContext` block too?

Reply:
@fabioCollini I'm not sure, not super familiar with Kotlin and coroutines yet.
I did it just because the IDE told me "Inappropriate blocking method call" otherwise :) 
write is already a suspend function

@digitalbuddha I have signed the cla!

Reply:
let's leave it I'll add to write as well

Reply:
@digitalbuddha I don't think you should move `write` in to `withContext`, it won't change anything. That write method is already suspended, so it already know in what context it should be run in.